apps/src/dance/DanceAiModal.tsx

```diff
@@ -64,6 +67,7 @@ interface DanceAiProps {
 }
 
 const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
+  console.log(DanceAiClient('cupcake,cry,demon'));
```

Comment:
Is this intentional?  We should be using the emoji selected by the user.

Reply:
This is a dummy test, it does take in the actual value later and outputs the log. Probably a good idea to delete this for the final version regardless